packages/compass-e2e-tests/tests/smoke.test.js

```diff
@@ -82,7 +59,7 @@ describe('Compass', function () {
       }
       await compass.client.connectWithConnectionForm(
         atlasConnectionOptions,
-        30_000
+        30000
```

Comment:
Linting didnt like it. I assume this still means the same thing?

Reply:
Yeah, although it’s unfortunate that the linter doesn’t recognize it :/

Reply:
We could also just update the common linting rules?

Reply:
If that's possible, let's do that -- but don't feel like you have to. :)